azurerm/provider.go

```diff
@@ -124,6 +124,8 @@ func Provider() terraform.ResourceProvider {
 
 			"azurerm_application_insights": resourceArmApplicationInsights(),
 
+			//PTR record uses Azure Go SDK
```

Comment:
_minor_: can we move this sorted alphabetically under the Azure SDK for Go comment?